src/test/suite/fixtures/recorded/actions/clearSource.yml

```diff
@@ -0,0 +1,30 @@
+languageId: plaintext
```

Comment:
This test is just testing "source", right? I'd put it in the `marks` dir

Reply:
Well it's specifically testing the source implementation of this specific action. If you want any move it

Reply:
Not really. The test for the action includes the "source" output already

Reply:
So it does. I will remove these tests

Reply:
Worth a quick double check to make sure that we have at least one test for the "source" mark in `marks` but otherwise yeah I'd just remove these